apps/homeless-people-help-map/src/translations/en.ts

```diff
@@ -15,6 +15,7 @@ export default {
     point: "Point",
     districtBorder: "District border",
     drinkingFountain: "Drinking fountain",
+    publicToilet: "Public toilet",
```

Comment:
Consider renaming all multiple word names in English with capital letter at the start of each word, such as "Public Toilets" or "Drinking Fountain". Not urgent, maybe for later after discussion.